index.html

```diff
@@ -0,0 +1,141 @@
+<!--Mobile version of template 1-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Mobile Version</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--Work section-->
+    <section class="work">
+        <!---The first card -->
+        <div class="flex">
+            <img src="images/card1-image.png" alt="Image for Card 1">
```

Comment:
- Why is the header section removed? Please make the project complete without removing parts of the project.


[OPTIONAL] Your css style of the previous projects are still not according to figma design. Please try to match the figma design in font, font-size, color, margin and padding.